client/my-sites/site-settings/form-general.jsx

```diff
@@ -858,19 +862,30 @@ export class SiteSettingsFormGeneral extends Component {
 	}
 
 	advancedCustomizationNotice() {
-		const { translate, selectedSite, siteSlug } = this.props;
-		const upgradeUrl = `/plans/${ siteSlug }?plan=${ PLAN_PREMIUM }&feature=${ FEATURE_STYLE_CUSTOMIZATION }`;
+		const { translate, selectedSite, siteSlug, globalStylesOnPersonalExperiment } = this.props;
+		const upgradeUrl = `/plans/${ siteSlug }?plan=${
+			globalStylesOnPersonalExperiment ? PLAN_PERSONAL : PLAN_PREMIUM
+		}&feature=${ FEATURE_STYLE_CUSTOMIZATION }`;
 
 		return (
 			<>
 				<div className="site-settings__advanced-customization-notice">
 					<div className="site-settings__advanced-customization-notice-cta">
 						<Gridicon icon="info-outline" />
-						<span>
-							{ translate(
-								'Your site contains customized styles that will only be visible once you upgrade to a Premium plan.'
-							) }
-						</span>
+						{ globalStylesOnPersonalExperiment ? (
+							<span>
+								{ translate(
+									'Your site contains customized styles that will only be visible once you upgrade to a Personal plan.'
+								) }
+							</span>
+						) : (
+							<span>
+								{ ' ' }
+								{ translate(
+									'Your site contains customized styles that will only be visible once you upgrade to a Premium plan.'
+								) }{ ' ' }
+							</span>
+						) }
```

Comment:
Can this be simplified like this?

```suggestion
						<span>
							{ globalStylesOnPersonalExperiment
								? translate(
										'Your site contains customized styles that will only be visible once you upgrade to a Personal plan.'
								  )
								: translate(
										'Your site contains customized styles that will only be visible once you upgrade to a Premium plan.'
								  ) }
						</span>
						) }
```

Reply:
Sure